en/editor/project/index.md

```diff
@@ -0,0 +1,86 @@
+# Project Setting
+
+The Project Settings windows are available from Cocos Creator 3D’s main menu (Project > Project Setting) which includes all the settings related to your project. These settings will be saved in the project's `settings / packages` folder. If you need to synchronize project settings between different developers, this folder should be added in your source control system.
+
+## General
+
+![general](./index/general.jpg)
+
+### Default Canvas settings
+
+The default Canvas settings include design resolution and adapted screen width/height, which are used to specify the default design resolution value in Canvas when creating a new scene or Canvas component, as well as the `Fit Height, Fit Width` options.
+
+For more information, please refer to [Multi-resolution adaptation scheme](../ui-system/components/engine/multi-resolution.md).
```

Comment:
For more information, please refer to [Multi-resolution adaptation scheme](../ui-system/components/engine/multi-resolution.md).
-->
For more information, please refer to the [Multi-resolution adaptation scheme](../ui-system/components/engine/multi-resolution.md) documentation.

Reply:
You missed "the"
--> please refer to the